src/main/java/xyz/nucleoid/extras/integrations/relay/RemoteCommandIntegration.java

```diff
@@ -35,7 +34,22 @@ public static void bind(NucleoidIntegrations integrations, IntegrationsConfig co
             integrations.bindReceiver("command", body -> {
                 var command = body.get("command").getAsString();
                 var sender = body.get("sender").getAsString();
-                integration.commandQueue.add(new RemoteCommand(command, sender));
+                var discordRoles = body.get("roles").getAsJsonArray();
+                var roles = new ArrayList<String>();
+                roles.add("discord_everyone");
```

Comment:
I didn't go with `everyone` because we probably don't want people on Discord to be able to do things like `/help` as it spams the channel and hits the send rate-limits because each line is a separate message